### PR TITLE
Added a few notes that may help the next person

### PR DESCRIPTION
**Pull request by Warren-Johnson-TID**
I added a few notes to the linux/README.md that may help the next person attempting to run this locally.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208333762429301

### CODE REVIEW

File: linux/README.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ You'll also need to put the Vault password (stored in 1Password) into
 
 ## Virtualbox 
 
-So far, this only works on an Intel Mac. It would probably work on an Intel Linux machine as well.
+~~So far, this only works on an Intel Mac. It would probably work on an Intel Linux machine as well.~~
+
+Update: VirtualBox in combination with Vagrant currently (September 2024) having issues starting on MBTA-issued Mac hardware.  It is suggested to utilize QEMU.
```
Comment (paulswartz) [inline linux/README.md:29]: question: do you have an Intel Mac? If it was issued this year, I'm pretty sure we've only been issuing Apple Silicon hardware.

File: linux/README.md [hunk 2]
```diff
@@ -52,7 +55,14 @@ $ ssh -p2222 <username>@localhost
 
 ## QEMU
 
-QEMU will work on an M1.
+QEMU will work on an M1.  
+
+Things learned the hard-way:
+
+*hostname* taken from inventory.yml file.  You can add any user manually by duplicating a user block in linux/cloud-init/group_vars.  This is useful if Ansible isn't running (and thus not adding your user).  If you're running qemu with no specific hostname, you would add the user in local/.
```
Comment (paulswartz) [inline linux/README.md:62]: suggestion: QEMU runs with the default hostname of `qemu` which is already in the inventory. Can you say more about the issue you ran into here?

Comment (paulswartz) [inline linux/README.md:62]: suggestion: the note about the "user block" isn't really QEMU-specific; maybe it should move to the Ansible section?

File: linux/README.md (hunk 1 above)
Comment (Warren-Johnson-TID) [reply linux/README.md:29]: I have an ARM processor.  I assume that's Apple Silicon Hardware?

Comment (paulswartz) [reply linux/README.md:29]: Probably; if you go the 🍎 menu and click "About", it'll tell you what specific chip you have.

**Pull request closed without merging**